extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -702,9 +701,15 @@ export class VisibilityManagerForEmbed extends VisibilityManager {
     return this.backgroundedAtStart_;
   }
 
-  /** @override */
+  /**
+   * Gets the layout box of the embedded document. Note that this may be
+   * smaller than the size allocated by the host. In that case, the document
+   * will be centered, and the unfilled space will not be reflected in this
+   * return value.
+   * @override
+   */
```

Comment:
nit: Could you also document that `embed.iframe` is used to calculate the root layoutbox. `embed.host` is used to calculate the root visibility. And the reasons. Thanks.

Reply:
Done, though please confirm my reasoning in the final sentence?